opte-api/src/ip.rs

```diff
@@ -524,6 +543,10 @@ impl Ipv6Addr {
     pub fn safe_mask(self, mask: Ipv6PrefixLen) -> Self {
         self.mask(mask.val()).unwrap()
     }
+
+    pub const fn from_const(bytes: [u8; 16]) -> Self {
```

Comment:
Ah, I remember why I didn't do that. It's pretty verbose, since you need to call `u16::to_be_bytes()` on each word in that array. But I agree it's probably worth it. I'm adding it now.